crates/ibc/src/core/ics26_routing/msgs.rs

```diff
@@ -36,7 +38,7 @@ impl TryFrom<Any> for MsgEnvelope {
                     .map_err(RouterError::MalformedMessageBytes)?;
                 Ok(MsgEnvelope::Client(ClientMsg::CreateClient(domain_msg)))
             }
-            update_client::UPDATE_CLIENT_TYPE_URL => {
+            update_client::TYPE_URL => {
                 let domain_msg =
                     <update_client::MsgUpdateClient as Protobuf<RawMsgUpdateClient>>::decode_vec(
                         &any_msg.value,
```

Comment:
Can we write `update_client::MsgUpdateClient::decode_vec(&any_msg.value)` instead?

Reply:
[60ff485](https://github.com/cosmos/ibc-rs/pull/643/commits/60ff485ee374d9be61e75fcf249c5f6a1f0eab29)